releaseNotes/master.md

```diff
@@ -6,6 +6,29 @@ ${RES_VER_DATE}
 ## Features
   - **Failure logs in email notifications**: Email notifications for both CI and Assembly Lines can be configured to include log output of failing commands with the `sendFailingSnippet` option. To enable this, see the [CI email](http://docs.shippable.com/ci/email-notifications/) or [Assembly Lines configuration](http://docs.shippable.com/platform/workflow/config/#assembly-lines-configuration) documentation.
 
+  - **Addons are enabled by default in fresh installation of Shippable** - After a [fresh installation of Shippable](http://docs.shippable.com/platform/server/install-onebox/), the following Addons will be enabled by default.
```

Comment:
- this needs to go in shippable server section
- each addon should be linked to respective docs

Reply:
ok. Thanks. Will update this PR.